etrago/appl.py

```diff
@@ -75,6 +75,7 @@
     # Scenario variations:
     "scn_extension": None,  # None or array of extension scenarios
     "scn_decommissioning": None,  # None or decommissioning scenario
+    "H2_vol_share": 15,  # in % [50/20/15/10/5/2/1/0] allowed H2 volumetric share for feedin
```

Comment:
Could you also add this parameter to the json file?

Reply:
Yes (https://github.com/openego/eTraGo/pull/602/commits/61d31a42699485681e33d620077ce15845e3cf36), sorry about forgetting this again!